content/_index.md

```diff
@@ -7,9 +7,39 @@ github_org= "https://github.com/valkey-io"
 github_org_text= "All repos in the Valkey GitHub organization"
 
 headline= "Valkey: an open source, in-memory data store"
-
 long_description= ""
 
+[extra.hero]
+headings = ["FAST.", "RELIABLE.", "OPEN SOURCE, FOREVER."]
+subtitle = "Valkey is an open source (BSD) high-performance key/value datastore that supports a variety of workloads such as caching, message queues, and can act as a primary database. The project is backed by the Linux Foundation, ensuring it will remain open source forever.
```

Comment:
```suggestion
subtitle = "Valkey is an open source (BSD) high-performance key/value datastore that supports a variety of workloads such as caching, message queues, and can act as a primary database. The project is backed by the Linux Foundation, ensuring it will remain open source forever."
```